lib/travis/api/v3/queries/subscriptions.rb

```diff
@@ -1,13 +1,23 @@
 module Travis::API::V3
   class Queries::Subscriptions < Query
+    params :plan, :coupon, :organization_id
+    params :first_name, :last_name, :company, :address, :address2, :city, :country, :state, :vat_id, :zip_code, :billing_email, prefix: :billing_info
+    params :card_owner, :expiration_date, :last_digits, prefix: :credit_card_info
+
     def all(user_id)
       client = BillingClient.new(user_id)
       client.all
     end
 
     def create(user_id)
       client = BillingClient.new(user_id)
-      client.create_subscription(params)
+      client.create_subscription(
+        :plan => params['plan'],
```

Comment:
If I remember rightly, there will also be autogenerated methods for these params too. So you can just do `:plan => plan` etc.

Reply:
Maybe you don't remember rightly :laughing: because that seems not to work and breaks the tests. It would have been cool.